ansible/tasks/migrate-db-name.yml

```diff
@@ -1,44 +0,0 @@
----
-- name: Aeternity binary exists
-  stat:
-    path: "{{ aeternity_bin }}"
-  register: binary
-  changed_when: false
-
-- name: Ping aeternity node
-  command: "{{ aeternity_bin }} ping"
-  register: ping
-  failed_when: no
-  changed_when: false
-  when: binary.stat.exists
-
-- name: Aeternity db exists
-  stat:
-    path: "{{ db_path }}/mnesia"
-  register: db
-  changed_when: false
-
-- name: Check if already migrated
-  find:
-    paths: "{{ db_path }}/mnesia"
-    patterns:
-      - "schema.DAT.backup"
-      - "schema.DAT.*.backup"
-  register: find_results
-  when:
-    - db.stat.exists
-
-- name: Stop aeternity node
-  command: "{{ aeternity_bin }} stop"
-  when:
-    - binary.stat.exists
-    - ping.stdout == "pong"
-    - db.stat.exists
-    - find_results.files|length == 0
-
-- name: Migrate db schema names
-  command: "{{ aeternity_bin }} rename_db {{ db_path }}"
```

Comment:
Shall we remove in the next major release [the script for renaming](https://github.com/aeternity/aeternity/blob/a6ab0c6dd7b9b25ae1ce8af271bbdf98cd2cf52a/rebar.config#L150-L163.)

Current node release supports [2.2.0+](https://github.com/aeternity/aeternity/blame/a6ab0c6dd7b9b25ae1ce8af271bbdf98cd2cf52a/docs/release-notes/RELEASE-NOTES-3.0.0-rc.1.md#L65) and rename was in [2.2.0](https://github.com/aeternity/aeternity/blame/a6ab0c6dd7b9b25ae1ce8af271bbdf98cd2cf52a/docs/release-notes/RELEASE-NOTES-2.2.0.md#L10).

Reply:
This repository while public is intended for internal usage, and we don't have any 2.2.0 already.

In regards to the node script, yes it makes sense to remove it in next major.